### PR TITLE
Adding usage and setup guide for forks and git

### PR DESCRIPTION
**Pull request by DasVoelkel**
also adding lots of pictures for clarity

### CODE REVIEW

**Comment by Veradux** [comment]
Is it normal that images don't show properly for me in the preview? They don't load.

File: docs/guides/guide_install_VSC.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+## Getting the required software
+Generally for developing you almost always use an IDE.
+You are free to develop in any IDE you like.
+
+### getting an IDE
+#### getting VSC (Visual Studio Code)
+Download VSCode [here](https://code.visualstudio.com/#alt-downloads).
+Download VSC for your appropriate system.
+
+### getting git
```
Comment (NishantAcharya) [inline docs/guides/guide_install_VSC.md]: Github Desktop is a nice Application, its the same thing as Git, but a bit less terminal based and can co-exist alongside git

**Comment by RedRoyalDog** [comment]
@DasVoelkel Can you make the changes @anthonyLuttrell has suggested?

@Plat251 Can you take point on this?

**Comment by Plat251** [comment]
@RedRoyalDog 
There's some re-writing that needs to be done, but I'll see what I can do.

**Comment by RedRoyalDog** [comment]
> @RedRoyalDog
> There's some re-writing that needs to be done, but I'll see what I can do.

If you get the rewriting done, or have OP rewrite it. I'll merge it.

**Comment by Plat251** [comment]
With a fresh look at this, I don't think that putting these guides through PRs and into the code is a worthy idea - Wiki exists, and not everyone who is the target audience will benefit, as they're not guaranteed to look for this in code.

**Comment by Plat251** [comment]
This has been integrated into Wiki with some editing. Further editing is necessary, though.

**Pull request closed without merging**